File: demos/hello-c/hello.c
#include <riv.h>

int main() {
    do {
        // clear screen
        riv_clear(RIV_COLOR_BLACK);
        // draw hello world
        riv_draw_text("hello world!", RIV_SPRITESHEET_FONT_5X7, RIV_CENTER, 128, 128, 2, RIV_COLOR_WHITE);
    } while(riv_present());
}

File: demos/hello-zig/hello.zig
const riv = @cImport({
    @cInclude("riv.h");
});

pub fn main() !void {
    while (riv.riv_present()) {
        riv.riv_clear(0);
        _ = riv.riv_draw_text("hello world!", riv.RIV_SPRITESHEET_FONT_5X7, riv.RIV_CENTER, 128, 128, 2, riv.RIV_COLOR_WHITE);
    }
}
